jobs/haproxy/spec

```diff
@@ -672,8 +672,27 @@ properties:
       Prefer backend servers which are located on the same availability zone. Note that this only affects servers provided via the http_backend link property. Servers provided via the tcp backend_link will automatically prefer the local AZ.
     default: false
   ha_proxy.enable_http2:
-    description: Enables ingress (frontend) and egress (backend) HTTP/2 ALPN negotiation. Egress (backend) HTTP protocol version may be overriden by `ha_proxy.backend_ssl`, `ha_proxy.disable_backend_http2_websockets` and `ha_proxy.backend_match_http_protocol`.
+    description: Enables ingress (frontend) and egress (backend) HTTP/2 ALPN negotiation. Egress (backend) HTTP protocol version may be overridden by `ha_proxy.backend_ssl`, `ha_proxy.disable_backend_http2_websockets` and `ha_proxy.backend_match_http_protocol`.
     default: false
   ha_proxy.always_allow_body_http10:
     description: Always allow a body to be sent when using HTTP/1.0. By default HAProxy denies GET/HEAD/DELETE requests with a body when using HTTP/1.0 due to potential request smuggling attacks. See https://github.com/haproxy/haproxy/commit/e136bd12a32970bc90d862d5fe09ea1952b62974
     default: false
+  
+  ha_proxy.requests_rate_limit.requests:
+    description: How many requests are allowed in the given time window from one IP address.
+  ha_proxy.requests_rate_limit.window_size:
+    description: Window size for counting requests.
+  ha_proxy.requests_rate_limit.table_size:
+    description: Size of the stick table in which the IPs and counters are stored.
+  ha_proxy.requests_rate_limit.block:
+    description: Whether or not to block requests.
+    default: false
```

Comment:
There is no test combination where we define limits but disable blocking, right? Maybe that would still be relevant, i.e. that if blocking is off every request still passes and none are blocked?